modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/resources/META-INF/resources/CheckboxMultiple/CheckboxMultiple.es.js

```diff
@@ -70,10 +70,12 @@ const CheckboxMultiple = ({
 	predefinedValue,
 	value: initialValue,
 }) => {
-	const [value, setValue] = useState(initialValue);
+	const [value, setValue] = useState(initialValue ?? predefinedValue);
 
 	useEffect(() => {
-		setValue(initialValue);
+		if (initialValue?.length > 0) {
```

Comment:
I see, to prevent regression we need to add this validation in FE.

@marcelabc, can we create a BE task to check the effort of returning the expected result after evaluating this rule?

This would allow us to remove these two changes in FE in the future, and should prevent similar errors in other fields.

Reply:
We decided to create a spike to analyze the effort required to solve this on the backend.
For now, we'll move forward with this frontend solution :smile: